bert_e/templates/request_integration_branches.md

```diff
@@ -0,0 +1,15 @@
+{% extends "message.md" %}
+
+{% block title -%}
+Request integration branches
+{% endblock %}
+
+{% block message %}
+Waiting for integration branch creation to be requested by the user.
+
+To request integration branches, please comment on this pull request with the following command:
```

Comment:
What about the fact we wanted other comments to be able to set this value to true?

Reply:
This message is displayed for people who need to type this command. In the case where someone writes `approve` at the beginning of their PR, they will instead receive a predefined message template that says: 'Integration branches have been created, congratulations! Here they are: ... blablabla.' They will never see this specific template, so I don't think it's necessary to mention it.